functions/openhab.bash

```diff
@@ -63,11 +78,13 @@ Check the \"openHAB Release Notes\" and the official announcements to learn abou
   if ! cond_redirect apt-get ${APT_INST_OPTS} install "openhab2=${openhabVersion}"; then echo "FAILED (apt)"; exit 1; fi
   cond_redirect adduser openhab gpio
   cond_redirect systemctl daemon-reload
+  cp "${BASEDIR}"/includes/start-level.script "${OPENHAB_USERDATA}"/etc/
```

Comment:
this is not active, right?
It could be implemented as an alternative approach compared to the renaming you added via systemd.
But I think it is not intended to be touched, and the different bundle names seem to change slightly between different OH releases.
So do we need this here?

Reply:
(this is a lengthy discussion you weren't involved in, please don't act on fractional knowledge)
Yes it *is* active. Yes it's an alternative approach but not comprehensive that's why I keep the delayed rules function active, too. They complement each other to some extent.
It is not intended to be touched _by users_ but for us it's the right place/way to enforce that.
No names don't change between releases (... any more)
No it does not do harm even if names should change again some day.

Reply:
I'd like to understand it when I to a review. 
1) Using the script for startup levels: I tried to do this on my installation, copied over the file to `/var/lib/openhab2/etc`, restarted OH. But `bundle:list` did not show any difference after restart. It should, right? This is why I asked if this is active.
2) Ok, if we do both, then please add a comment. For me it looked like two independent approaches for the same problem.

Reply:
It worked on my test box. You must place it in `/var/lib/openhab2/etc`**scripts/**